pkg/test/framework/components/echo/kube/instance.go

```diff
@@ -643,7 +643,7 @@ func (c *instance) Restart() error {
 			return fmt.Errorf("expected %d pods, got %d", len(origPods), len(pods))
 		}
 		return nil
-	}, retry.Delay(time.Second*2), retry.Timeout(time.Second*30))
```

Comment:
Can you make this a constant defined at the top? Ideally, it would be nice to have it controlled by a flag as well so that we can deal with extra latency introduced when deploying to a real cloud